### ISSUE TITLE
05e6d7f2bddb0 Bleedingedge - Gray mission decline

### ISSUE THREAD

**Issue by Pluckyduck** [ISeeDEDPpl/Questor#314]
The Wildcat Strike is on the grey list and does not get turned down.  If i add it to the Black list it gets killed off.  Log below

09:14:09 [AgentInteraction] Waiting for mission
09:14:21 [AgentInteraction] [Coooliest Mug] standing toward me is [9.69], minAgentGreyListStandings: [5]
09:14:21 [AgentInteraction] There are No Bookmarks Associated with The Wildcat Strike yet

**Comment by br3sson** [comment]
There's a problem in the Setting.cs about the spelling of "grey", actually questor is ignoring cause he search for a blocks of grAylist not grEylist

Just change greylist with graylist in your xml block. sorry cannot insert the < > here :-/

Relevant commit:
https://github.com/ISeeDEDPpl/Questor/commit/88aa0c50e28c2b031de3d1cb54dc5aa480600867
FIle: Settings.cs
Line: 1308

**Comment by Pluckyduck** [comment]
thanks

**Comment by Pluckyduck** [comment]
So far, q has started declining the mission again on the list.  Thank for the find.

**Comment by ISeeDEDPpl** [comment]
I fixed this in correct bleedingedge, for consistency, and to fix it for everyone who did not know about this issue, I changed the code to look for greylist, all the other occurrences were greylist not containing an "a".

If you have modified your XML you will need to change it back after updating to bleedingedge. For clarity, the settings.xml in the output directory has it correct now and should be sync'd with (winmerge?) as needed.
